src/openstreetmap-uk.js

```diff
@@ -44,40 +44,24 @@ D(DOMAIN, REGISTRAR, DnsProvider(PROVIDER),
 
   // Aerial imagery sites
 
-  A("hampshire.aerial", IPV4["lockheed"]),
-  AAAA("hampshire.aerial", IPV6["lockheed"]),
-  A("a.hampshire.aerial", IPV4["lockheed"]),
-  AAAA("a.hampshire.aerial", IPV6["lockheed"]),
-  A("b.hampshire.aerial", IPV4["lockheed"]),
-  AAAA("b.hampshire.aerial", IPV6["lockheed"]),
-  A("c.hampshire.aerial", IPV4["lockheed"]),
-  AAAA("c.hampshire.aerial", IPV6["lockheed"]),
-
-  A("surrey.aerial", IPV4["lockheed"]),
-  AAAA("surrey.aerial", IPV6["lockheed"]),
-  A("a.surrey.aerial", IPV4["lockheed"]),
-  AAAA("a.surrey.aerial", IPV6["lockheed"]),
-  A("b.surrey.aerial", IPV4["lockheed"]),
-  AAAA("b.surrey.aerial", IPV6["lockheed"]),
-  A("c.surrey.aerial", IPV4["lockheed"]),
-  AAAA("c.surrey.aerial", IPV6["lockheed"]),
-
-  A("os", IPV4["lockheed"]),
-  AAAA("os", IPV6["lockheed"]),
-  A("a.os", IPV4["lockheed"]),
-  AAAA("a.os", IPV6["lockheed"]),
-  A("b.os", IPV4["lockheed"]),
-  AAAA("b.os", IPV6["lockheed"]),
-  A("c.os", IPV4["lockheed"]),
-  AAAA("c.os", IPV6["lockheed"]),
-
-  A("ea", IPV4["lockheed"]),
-  AAAA("ea", IPV6["lockheed"]),
-  A("a.ea", IPV4["lockheed"]),
-  AAAA("a.ea", IPV6["lockheed"]),
-  A("b.ea", IPV4["lockheed"]),
-  AAAA("b.ea", IPV6["lockheed"]),
-  A("c.ea", IPV4["lockheed"]),
-  AAAA("c.ea", IPV6["lockheed"])
+  osm_web_service("hampshire.aerial",   "lockheed"),
```

Comment:
Do we want to align the second arguments like this? It's not consistent with any of the other records...

Reply:
To me it looks like it correctly aligns. Maybe I am missing something.

Reply:
It correctly aligns yes, but nowhere else in the file do we add spaces to make the second argument line up?

Reply:
I do align the 2nd argument up in this and other files. I find it more readable, but happy to flatten it if you prefer that.

Reply:
All flattened now.